mods/ra/maps/evacuation/map.yaml

```diff
@@ -0,0 +1,1757 @@
+MapFormat: 11
+
+RequiresMod: ra
+
+Title: Evacuation
+
+Author: Scott_NZ
+
+Tileset: SNOW
+
+MapSize: 128,128
+
+Bounds: 16,16,96,96
+
+Visibility: MissionSelector, Lobby
```

Comment:
Having a thing appear in two different places that otherwise display completely disjoint content is terrible UX.